Common/Statistics/StatisticsBuilder.cs

```diff
@@ -221,6 +221,8 @@ public static class StatisticsBuilder
                 { PerformanceMetrics.CompoundingAnnualReturn, Math.Round(totalPerformance.PortfolioStatistics.CompoundingAnnualReturn.SafeMultiply100(), 3).ToStringInvariant() + "%" },
                 { PerformanceMetrics.Drawdown, Math.Round(totalPerformance.PortfolioStatistics.Drawdown.SafeMultiply100(), 3).ToStringInvariant() + "%" },
                 { PerformanceMetrics.Expectancy, Math.Round(totalPerformance.PortfolioStatistics.Expectancy, 3).ToStringInvariant() },
+                { PerformanceMetrics.StartingEquity, Math.Round(totalPerformance.PortfolioStatistics.StartingEquity, 3).ToStringInvariant() },
+                { PerformanceMetrics.EndingEquity, Math.Round(totalPerformance.PortfolioStatistics.EndingEquity, 3).ToStringInvariant() },
```

Comment:
Nice! Should review `OptimizationBacktestJsonConverter` since these new values will impact there too, think we should be able to skip them, like the sortino ratio